Tests/ML/histopathology/preprocessing/test_slide_loading.py

```diff
@@ -130,6 +139,7 @@ def test_get_bounding_box(level: int, foreground_threshold: Optional[float]) ->
 @pytest.mark.parametrize('level', [1, 2])
 @pytest.mark.parametrize('margin', [0, 42])
 @pytest.mark.parametrize('foreground_threshold', [None, 215])
+@pytest.mark.skipif(is_windows(), reason="cucim package is not available on Windows")
 def test_load_roi(level: int, margin: int, foreground_threshold: Optional[float]) -> None:
     dataset = MockSlidesDataset()
     sample = dataset[0]
```

Comment:
is this moving the tests to Windows and skipping them a temporary workaround?
skipping all tests related to cucim is a problem for the histo pipeline. For now these functions are used less frequently because they are used for patching and we store the patches. But if we want to move towards online patching this will prevent testing on the end-to-end pipeline